app/api/discount_codes.py

```diff
@@ -332,15 +329,14 @@ def before_get_object(self, view_kwargs):
             view_kwargs['event_id'] = event.id
 
         if view_kwargs.get('event_id') and has_access('is_admin'):
-            event = safe_query(self, Event, 'id', view_kwargs['event_id'], 'event_id')
+            event = safe_query(Event, 'id', view_kwargs['event_id'], 'event_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)